scripts/buildimages.sh

```diff
@@ -3,11 +3,13 @@ set -e
 
 source scripts/runsetup.sh
 
-MAIN_IMAGE=${DJANKISERV_DOCKER_REPO}/djankiserv:$(git describe --tags)
-docker build -f images/main/Dockerfile . -t ${MAIN_IMAGE}
-docker push ${MAIN_IMAGE}
-
-STATIC_IMAGE=${DJANKISERV_DOCKER_REPO}/djankiserv-static:$(git describe --tags)
-python src/manage.py collectstatic --noinput
-docker build -f images/static/Dockerfile . -t ${STATIC_IMAGE}
-docker push ${STATIC_IMAGE}
+TAG=${DJANKISERV_BUILD_TAG:-$(git describe --tags)}
+
+MAIN_IMAGE=${DJANKISERV_DOCKER_REPO}/djankiserv:$TAG
+buildah bud -t ${MAIN_IMAGE} -f images/main/Dockerfile .
```

Comment:
This is only a minor, but I usually write [this snippet](https://stackoverflow.com/a/246128) at the top of a `bash` script to get the directory of itself:

```
DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
```

...and then I reference the hard-coded files from there, like `images/main/Dockerfile`. Otherwise this setup assumes that I'm in a specific directory.

Reply:
@kuklinistvan , yeah, that has advantages but for very specific stuff I often don't bother. When you are talking about a script that can be launched in lots of different environments I definitely agree but this is only supposed to occasionally be launched by users and 99% launched in a single CI/CD environment (which doesn't exist yet :-)). I'll think about it for later.